src/Seq.App.EmailPlus/EmailApp.cs

```diff
@@ -132,18 +145,15 @@ public void On(Event<LogEventData> evt)
 
             var to = FormatTemplate(_toAddressesTemplate.Value, evt, base.Host);
             var body = FormatTemplate(_bodyTemplate.Value, evt, base.Host);
-            var subject = FormatTemplate(_subjectTemplate.Value, evt, base.Host).Trim().Replace("\r", "").Replace("\n", "");
+            var subject = FormatTemplate(_subjectTemplate.Value, evt, base.Host).Trim().Replace("\r", "")
+                .Replace("\n", "");
             if (subject.Length > MaxSubjectLength)
                 subject = subject.Substring(0, MaxSubjectLength);
 
-            var client = new SmtpClient(Host, Port ?? 25) {EnableSsl = EnableSsl ?? false};
-            if (!string.IsNullOrWhiteSpace(Username))
-                client.Credentials = new NetworkCredential(Username, Password);
+            var result = await _mailGateway.Send(_client, _options, new MimeMessage(new List<InternetAddress> {MailboxAddress.Parse(From)},
+                new List<InternetAddress> {MailboxAddress.Parse(to)}, subject,
+                (new BodyBuilder() {HtmlBody = body}).ToMessageBody()));
```

Comment:
Looks like we no longer set credentials for the outgoing mail server

Reply:
@nblumhardt  We do, but it's in the SmtpOptions that are set during OnAttached.

Reply:
It's set on `_options`, but never passed to the client.

Reply:
Well, crap. Will fix.

Reply:
Missing code found down the back of the sofa